src/orion/client/__init__.py

```diff
@@ -209,10 +205,10 @@ def build_experiment(
             "max_idle_time is deprecated. Use experiment.workon(reservation_timeout) instead."
         )
 
-    setup_storage(storage=storage, debug=debug)
+    builder = experiment_builder.ExperimentBuilder(storage, debug)
```

Comment:
Docstring must be adjusted accordingly.

Reply:
Ah, no, you are still passing the configuration. Sorry, forget about my comment above.